packages/babel-preset-env/data/built-ins.json

```diff
@@ -1,4 +1,204 @@
+    "electron": "1.1"
+  },
+  "es6.array.for-each": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.index-of": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.is-array": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "4",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.last-index-of": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.map": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.reduce": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "3",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.reduce-right": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "3",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.some": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.sort": {
+    "chrome": "63",
+    "opera": "50",
+    "edge": "12",
+    "firefox": "5",
+    "safari": "10.1",
+    "ie": "9",
+    "ios": "10.3"
+  },
+  "es6.date.now": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.date.to-iso-string": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "3.5",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.date.to-json": {
+    "chrome": "5",
+    "opera": "12.1",
+    "edge": "12",
+    "firefox": "4",
+    "safari": "10",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "10",
+    "electron": "1.1"
+  },
+  "es6.function.bind": {
+    "chrome": "7",
+    "opera": "12",
+    "edge": "12",
+    "firefox": "4",
+    "safari": "5.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "7"
+  },
+  "es6.string.iterator": {
```

Comment:
I'm pretty sure this doesn't count as "ES5 support" :)

But it's good that this is being fixed; I'm explicitly importing `core-js/fn/string/iterator` to work around it.